estate/models/estate_property.py

```diff
@@ -0,0 +1,36 @@
+from odoo import fields, models
+
+import datetime
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Property of the estate"
+
+    name = fields.Char('Name', required=True)
+    description = fields.Text('Description')
+    postcode = fields.Char('Postcode')
+    date_availability = fields.Date('Availability Date', copy=False, default=lambda _: fields.Date.today()+datetime.timedelta(days=90))
```

Comment:
```suggestion
    date_availability = fields.Date('Availability Date', copy=False, default=fields.Date.add(fields.Date.today(), months=3))
```
We can also do it like this.